components/navbar/ProfileDropdown.tsx

```diff
@@ -42,10 +42,11 @@ const DropdownCard = (props: RoleNavbarProps & DropdownCardProps) => {
 }
 
 const ProfileDropdown = (props: RoleNavbarProps) => {
+  const [open, setOpen] = React.useState(false)
```

Comment:
Import `useState` instead of `React.useState`

Reply:
Resolved in [87a094f](https://github.com/imperial/cpp-connect/pull/77/commits/87a094fd67d848a397f5210761b60ad0fc154c65)